### PR TITLE
Improve DurableExecutorService single run test

### PR DESCRIPTION
**Pull request by tkountis**

### CODE REVIEW

**Comment by tkountis** [comment]
Minor test change, review not needed.

File: hazelcast/src/test/java/com/hazelcast/durableexecutor/DurableRetrieveResultTest.java [hunk 1]
```diff
@@ -24,18 +24,14 @@
...
 import org.junit.runner.RunWith;
 
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.Future;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/durableexecutor/DurableRetrieveResultTest.java]: Please no star imports (I plan to activate CheckStyle for tests someday and those are not allowed)

Comment (tkountis) [inline hazelcast/src/test/java/com/hazelcast/durableexecutor/DurableRetrieveResultTest.java]: yup, new computer, need to fix my IDE ;)

Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/durableexecutor/DurableRetrieveResultTest.java]: The struggle is real! One of the stupid defaults from IntelliJ

**Pull request merged**